tests/views_widgets/BaseSnapshotTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use Spatie\Snapshots\MatchesSnapshots;
+
+class BaseSnapshotTest extends \Codeception\TestCase\WPTestCase {
+	use MatchesSnapshots;
+
+	/**
+	 * It should correctly test a string snapshot
+	 *
+	 * @test
+	 */
+	public function should_correctly_test_a_string_snapshot() {
+		uopz_set_return( 'time', '1577750400' );
```

Comment:
TEC is still *not* on `tric`, you cannot use a `uopz` function (yet).
This will alwasy fail in CI as Travis CI PHP version does not come with `uopz` installed.

If you really need it, then see [how to install additional PHP extensions in Travis CI](https://docs.travis-ci.com/user/languages/php/#installing-additional-php-extensions) and install it with `pecl install uopz`.

Reply:
Removed, It was a sample test.